spongehome.go

```diff
@@ -26,23 +26,68 @@
 package main
 
 import (
-    "github.com/SpongePowered/SpongeHome/controllers"
-    "github.com/go-macaron/pongo2"
-    "gopkg.in/macaron.v1"
+	"fmt"
+	"github.com/SpongePowered/SpongeHome/controllers"
+	"github.com/go-macaron/pongo2"
+	"github.com/sethvargo/go-fastly"
+	"gopkg.in/macaron.v1"
+	"os"
+	"time"
 )
 
+func clearFastly() {
+	// Clear Fastly cache
+	if os.Getenv("FASTLY_KEY") != "" {
+		time.Sleep(10 * time.Second) // Let the http server startup
+		fmt.Println("starting fastly client")
+		client, err := fastly.NewClient(os.Getenv("FASTLY_KEY"))
+		if err != nil {
+			fmt.Println(err)
+		}
+		services, err := client.ListServices(&fastly.ListServicesInput{})
+		fastlyName := ""
+		if os.Getenv("SPONGE_ENV") == "prod" {
+			fastlyName = "www.spongepowered.org"
+		} else if os.Getenv("SPONGE_ENV") == "staging" {
+			fastlyName = "www-staging.spongepowered.org"
+		}
+		if os.Getenv("SPONGE_ENV") == "" {
```

Comment:
Can you just flip this to an `else`? You probably want to `return` here too, having printed your message.